doc/man/scons.xml

```diff
@@ -2865,32 +2881,38 @@ to the Python module:</para>
 from SCons.Script import *
 </programlisting>
 
-<para>All builder methods return a list-like object
-containing Nodes that will be built.
-A <firstterm>Node</firstterm>
-is an internal SCons object
-which represents
-build targets or sources.
-See <xref linkend="node_objects"/> for more information.</para>
+<para>Builder methods return a <firstterm>Node-list</firstterm>,
```

Comment:
The object is actually named `NodeList` if you wanted to use that.

https://github.com/SCons/scons/blob/master/SCons/Node/__init__.py#L1708

Reply:
okay, sure - just copied from a bit of existing text in the manpage, rather use the correct term (search for `Node-list` in manpage, I had two hits in existing) - let me know.

Reply:
Oh - that leads me to repeat an earlier question: why are there two `NodeList` definitions?  The other is in `Util.py`.

Reply:
re multiple NodeList's that's a good question. The Util.py one has the functionality i'd normally expect from such.

Reply:
okay, so separate from this PR, maybe I'll do an experiment if switching Builder to the util-NodeList leaves everything working.